pandas/core/base.py

```diff
@@ -1338,8 +1338,8 @@ def factorize(self, sort=False, na_sentinel=-1):
 
         Parameters
         ----------
-        value : array_like
-            Values to insert into `self`.
+        value : scalar or array_like
+            Value(s) to insert into `self`.t
```

Comment:
```suggestion
             Value(s) to insert into ``self.t``.
```
I think render the whole `self.t` as code makes more sense (also, we end parameter descriptions with a period).

Reply:
I think this is actually better: ``Value(s) to search for in ``self``.``

As we're nor actually inserting, but searching.